misc/services/camel/de-metas-camel-shipping/src/main/java/de/metas/camel/shipping/shipment/ShipmentXmlToJsonRouteBuilder.java

```diff
@@ -36,7 +37,7 @@
 				.to(LOCAL_STORAGE_URL)
 				.streamCaching()
 				.unmarshal(jacksonXMLDataFormat)
-				.process(new ShipmentXmlToJsonProcessor())
+				.process(new ShipmentXmlToJsonProcessor()).id(SHIPMENT_XML_TO_JSON_PROCESSOR)
```

Comment:
hm, i think i wasn't aware of this :thinking: i'll try to keep it in mind

Reply:
used it for testing, basically to be able to add a mock node right after it that facilitates checking what the processor did